spec/features/audit/lists/filter_spec.rb

```diff
@@ -91,166 +90,222 @@
     let!(:audit) { create(:audit, content_item: felling) }
 
     scenario "filtering audited content" do
-      visit audits_path
-      select "Anyone", from: "allocated_to"
+      filter_audit_list = ContentAuditTool.new.filter_audit_list_page
+      filter_audit_list.load
 
-      choose "Audited"
-      click_on "Apply filters"
+      filter_audit_list.filter_form do |form|
+        form.allocated_to.select "Anyone"
+        form.audit_status.choose "Audited"
 
-      expect(page).to have_content("Tree felling")
-      expect(page).to have_no_content("Forest management")
-      expect(page).to have_checked_field("audit_status_audited")
+        form.apply_filters.click
+      end
+
+      expect(filter_audit_list.list).to have_content("Tree felling")
+      expect(filter_audit_list.list).to have_no_content("Forest management")
+      expect(filter_audit_list.filter_form.audit_status).to have_checked_field("audit_status_audited")
     end
 
     scenario "filtering for content regardless of audit status" do
-      visit audits_path
-      select "Anyone", from: "allocated_to"
+      filter_audit_list = ContentAuditTool.new.filter_audit_list_page
+      filter_audit_list.load
 
-      choose "All"
-      click_on "Apply filters"
+      filter_audit_list.filter_form do |form|
+        form.allocated_to.select "Anyone"
+        form.audit_status.choose "All"
 
-      expect(page).to have_content("Tree felling")
-      expect(page).to have_content("Forest management")
-      expect(page).to have_checked_field("audit_status_all")
+        form.apply_filters.click
+      end
+
+      expect(filter_audit_list).to have_content("Tree felling")
+      expect(filter_audit_list).to have_content("Forest management")
+      expect(filter_audit_list.filter_form.audit_status).to have_checked_field("audit_status_all")
     end
 
     context "when showing content regardless of audit status" do
-      before(:each) do
-        visit audits_path
-        select "Anyone", from: "allocated_to"
-
-        choose "All"
-        click_on "Apply filters"
-      end
-
       scenario "filtering by primary organisation" do
-        expect(page.find("#primary")).to be_checked
+        filter_audit_list = ContentAuditTool.new.filter_audit_list_page
+        filter_audit_list.load
 
-        select "HMRC", from: "Organisations"
+        filter_audit_list.filter_form do |form|
+          form.allocated_to.select "Anyone"
+          form.audit_status.choose "All"
+          form.apply_filters.click
 
-        click_on "Apply filters"
+          expect(form.primary_orgs).to have_checked_field
 
-        expect(page).to have_content("VAT")
-        expect(page).to have_no_content("Tree felling")
+          form.organisations.select "HMRC"
+          # select "HMRC", from: "Organisations"
```

Comment:
What are we alluding to with this comment that we can't infer from the line of code above it?